README.md

```diff
@@ -1,10 +1,46 @@
-GenAI contains pipelines that implement image and text generation tasks. The implementation exploits OpenVINO capabilities to optimize the pipelines. Each sample covers a family of models and suggests that its implementation can be modified to adapt for a specific need.
+<!---
+- [Repository Description](#repository-description))
+   - [Pipeline Samples](#pipeline-samples)
+   - [License](#license)
+- [Requirements](#requirements)
+-->
+## GenAI Pipeline Repository
 
-> [!NOTE]
-> This project is not for production use.
 
-The project includes the following pipelines:
+The GenAI repository contains pipelines that implement image and text generation tasks.
+The implementation uses OpenVINO capabilities to optimize the pipelines. Each sample covers
+a family of models and suggests certain modifications to adapt the code to specific needs.
+It includes the following pipelines:
 
 1. [Benchmarking script for large language models](./llm_bench/python/)
 2. [Casual LM](./text_generation/casual_lm/cpp/)
 3. [OpenVINO Stable Diffuison (with LoRA) C++ pipeline](./image_generation/stable_diffusion_1_5/cpp/)
+
+> [!NOTE]
+> This project is not for production use.
+
+### License
+
+The GenAI repository is licensed under [Apache License Version 2.0](LICENSE).
+By contributing to the project, you agree to the license and copyright terms therein and release
+your contribution under these terms.
+
+## Requirements
+
+Requirements may vary for different samples. See respective readme files for more details.
+To use the pipelines install a [proper OpenVINO version](docs.openvino.ai/install), 
```

Comment:
Every pipeline describes which version they require. Why would you introduce such an ambiguous instruction in root readme in first place? The same goes to `To build the pipelines, including the user_ov_extensions submodule, you can use commands like these:`

Reply:
As mentioned in the previous comment, it is to indicate that people need to check the readme file for the chosen sample to get the requirements. If there are any requirements common to all samples, they should be mentioned here. If there are no such requirements, we can just state "go and check readme files for each sample you want to use."
The info on the user_ov_extensions comes from the original content I used as the base. If you think it is irrelevant, let's remove it.

Reply:
Building user_ov_extensions is required only for C++ pipelines. My proposal is to keep "an OpenVINO version referred in the sample" general message, but remove build instruction from here.